src/signed_data.rs

```diff
@@ -598,6 +648,32 @@ mod tests {
                              "rsa-pss-sha256-salt10.pem",
                              Err(Error::UnsupportedSignatureAlgorithm));
 
+    /// Our PSS tests that should work.
+    test_verify_signed_data!(
+        test_rsa_pss_sha256_salt32,
+        "ours/rsa-pss-sha256-salt32.pem",
+        Ok(()));
+    test_verify_signed_data!(
+        test_rsa_pss_sha384_salt48,
+        "ours/rsa-pss-sha384-salt48.pem",
+        Ok(()));
+    test_verify_signed_data!(
+        test_rsa_pss_sha512_salt64,
+        "ours/rsa-pss-sha512-salt64.pem",
+        Ok(()));
+    test_verify_signed_data!(
+        test_rsa_pss_sha256_salt32_bad,
```

Comment:
Please name these something more specific than "bad", which indicates what the specific problem is.